fundingmanager.go

```diff
@@ -176,6 +177,25 @@ type numPendingReq struct {
 	resp chan uint32
 }
 
+// TODO: (andrew.shvv) Find better place for this function.
+// checkMaxPending check number of pending channels to not exceed maximum value
+// send the lnwire.ErrorGeneric message to the offending peer if the constraint
+// is violated.
+func (f *fundingManager) checkMaxPending(peer *peer) bool {
+	if len(f.activeReservations[peer.id]) >= maxActiveReservations {
+		errMsg := &lnwire.ErrorGeneric{
```

Comment:
This error shouldn't be unconditionally sent since in the case of a client using the `OpenChannel` RPC, the remote peer hasn't yet received any funding negotiation messages.